parking-lot-help-request/firmware/README.md

```diff
@@ -0,0 +1,99 @@
+# Parking Lot Help Request Firmware
+
+This project is meant to be host-free because the Notecard itself can be configured to send notes to Notehub when the help button is pressed.
+
+Instead of a true firmware set up guide, the [first part](#configure-the-notecard) of this document will guide you through creating a reusable Notecard configuration script which can be executed through the [Notecard CLI](https://dev.blues.io/tools-and-sdks/notecard-cli/) or through the [online webREPL](https://dev.blues.io/notecard-playground/) available on the Blues Developer Experience site.
+
+The [second section](#transform-notehub-data-with-jsonata-for-twilio), covers the data transformation that will need to happen inside of a Notehub project before a button press event sent to Notehub is routed out to Twilio to trigger an SMS alert to a user's phone.
+
+### Configure the Notecard
+
+There are two main ways to program a standalone Notecard that will not be interacting with a host microcontroller: 
+1. [The Notecard CLI](https://dev.blues.io/tools-and-sdks/notecard-cli/): a downloadable CLI designed for the Notecard.
+2. The [Notecard Playground](https://dev.blues.io/notecard-playground/): an in-browser terminal that emulates much of the functionality of the Notecard CLI. 
+
+Both of these tools are accessible through the Blues [Developer Experience site](https://www.dev.blues.io).
+
+> **NOTE**: If you _do_ choose to use the in-browser terminal, you'll need to do so on a Chrome-based web browser like Chrome or Edge that supports the web serial API. 
+
+![Screenshot of the in-browser Notecard Playground on dev.blues.io](./readme-notecard-playground.png)
+_The in-browser Notecard Playground available on dev.blues.io._
+
+The instructions to configure the Notecard will assume the Notecard CLI is being used, but this will also work with a very slight modification if you choose to use the in-browser Notecard Playground instead.
+
+1. Before you can configure your Notecard, you must set up a free [Notehub.io](https://notehub.io) account and create a project where the Notecard will send its data to. Follow the [Set Up Notehub quickstart](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-f/#set-up-notehub) to sign up and create your first project if you haven't done so already. Be sure to copy the Project UID after creating it - you'll need it for the Notecard configuration script.
```

Comment:
Not a big deal, but https://dev.blues.io/notehub/notehub-walkthrough/#create-a-notehub-account and https://dev.blues.io/notehub/notehub-walkthrough/#create-a-new-project are probably better links, as currently you’re linking to the Notecarrier-F’s quickstart.

Reply:
Much better links, thanks